daemon/test_support.go

```diff
@@ -6,18 +6,109 @@
 package daemon
 
 import (
+	"context"
+	"crypto/ecdsa"
 	"fmt"
+	"math/big"
 	"net"
+	"sync"
 	"syscall"
 	"testing"
 	"time"
 
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/stretchr/testify/require"
+
+	"github.com/athanorlabs/atomic-swap/common"
+	contracts "github.com/athanorlabs/atomic-swap/ethereum"
+	"github.com/athanorlabs/atomic-swap/ethereum/extethclient"
+	"github.com/athanorlabs/atomic-swap/monero"
+	"github.com/athanorlabs/atomic-swap/rpcclient"
+	"github.com/athanorlabs/atomic-swap/tests"
+)
+
+// map indexes for our mock tokens
```

Comment:
DAI has 18 decimals and USDT has 6, so it seemed like a good initial pair to be writing lots of tests against.